next.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   i18n: {
     // locales: ['en-us', 'ta-in', 'hi-in', 'kn-in'],
-    locales: ["en-us", "hi-in"],
+    locales: ["en-us", "hi-in", "gu-in"],
```

Comment:
*suggestion* 
can we pull language code list from a constant file.

Reply:
So this is supposed to be the constant file

This is how the next team recommends it

https://nextjs.org/docs/advanced-features/i18n-routing